src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -3030,6 +2872,9 @@ KeywordHandlers::KeywordHandlers()
         { "WTRACER" , &handleWTRACER    },
     };
 
+    for (const auto& it : getNetworkHandlers()) {
+        handler_functions.emplace(it.first, it.second);
+    }
     for (const auto& it : getUDQHandlers()) {
         handler_functions.emplace(it.first, it.second);
     }
```

Comment:
If there are going to be more of these, we **might** consider writing it as
```C++
for (const auto* handlerFactory : { &getNetworkHandlers, &getUDQHandlers }) {
    for (const auto& [keyword, handler] : std::invoke(handlerFactory)) {
        this->handler_functions.emplace(keyword, handler);
    }
}
```
instead.  Of course, if we do write it like that, then we're also forcing each handler factory function to have the same declaration which may or may not be too onerous.